chrome/content/zotero/fileInterface.js

```diff
@@ -129,7 +131,8 @@ var Zotero_File_Interface = new function() {
 	 */
 	function exportFile() {
 		var exporter = new Zotero_File_Exporter();
-		exporter.name = Zotero.getString("pane.collections.library");
+		exporter.libraryID = ZoteroPane_Local.getSelectedLibraryID();
```

Comment:
This could return false if something other than a library is selected (e.g. "Group Libraries" label). I don't think we should be able to get to this point in those circumstances, so you should throw if that's the case.